docs/guides/multisig-proposal.md

```diff
@@ -110,26 +125,25 @@ Let's go through each of the functions that are overridden.
 
 -   `name()`: Define the name of your proposal.
 -   `description()`: Provide a detailed description of your proposal.
--   `_deploy()`: Deploy any necessary contracts. This example demonstrates the
+-   `deploy()`: Deploy any necessary contracts. This example demonstrates the
     deployment of Vault and an ERC20 token. Once the contracts are deployed,
     they are added to the `Addresses` contract by calling `addAddress()`.
--   `_build()`: Set the necessary actions for your proposal. In this example,
+-   `run()`: Sets environment for running the proposal. It sets `addresses` and `primaryForkId`. `addresses` is address object
+    containing addresses to be used in proposal that are fetched from `Addresses.json`. `primaryForkId` is the RPC URL or alias of the blockchain that will be used to simulate the proposal actions and broadcast if any contract deployment is required.
+-   `build()`: Set the necessary actions for your proposal. In this example,
```

Comment:
I think `run` should come either at the beginning or at the end. In the beginning as the proposal environment is setup in run and in the end because it runs all the other functions. Should this be moved to the end or right after build.

Reply:
Done